distribution/src/main/resources/bin/plugin.bat

```diff
@@ -9,9 +9,46 @@ for %%I in ("%SCRIPT_DIR%..") do set ES_HOME=%%~dpfI
 
 TITLE Elasticsearch Plugin Manager ${project.version}
 
+SET properties=
+SET args=
+
+:loop
+SET "current=%~1"
+SHIFT
+IF "x!current!" == "x" GOTO breakloop
+
+IF "!current:~0,2%!" == "-D" (
```

Comment:
why do we need the distinction of -D and -D*=* ? Can we not handle it like here https://github.com/elastic/elasticsearch/blob/master/distribution/src/main/resources/bin/elasticsearch.bat ? I don't actually know a lot of bat so I am really just curious.